readme.md

```diff
@@ -11,7 +11,7 @@
 - ``:help pymode``
 - <https://github.com/python-mode/python-mode/wiki>
 
-**Please use python-mode tag on Stackoverflow to ask questions:**  
+**Please use python-mode tag on Stackoverflow to ask questions:**
```

Comment:
This line and a few others below had unnecessary trailing whitespace on the line.